addon/components/search-facet-worktype-button/template.hbs

```diff
@@ -0,0 +1,12 @@
+{{!Copied from Ember-SHARE}}
+<div class='type-filter-option {{if selected 'active'}}'>
+    {{#if collapsible}}
+        <i class='fa expand-icon clickable {{if toggleState 'fa-plus' 'fa-minus'}}' {{action 'toggleBody'}}></i>
+    {{/if}}
+    <span class={{if selected 'bubble'}}>
+        <a class='clickable' {{action 'click'}}>{{label}}</a>
```

Comment:
Consider changing all interactive elements in this file to `buttons` instead of `a` or `i` elements.

As elsewhere, I would also recommend adding `aria-labels` to at least some of the elements in this file, so they are not invisible to screen readers.